### PR TITLE
repositories and seed

### PR DESCRIPTION
**Pull request by K-Karalius**
- added repositories to manipulate deck and card data in the database 
- added seed class to populate the database for testing
- fixed a minor error in the db context and fixed the problems with namespaces
- updated readMe (how to populate the database with the seed)

### CODE REVIEW

File: aiKart/Repositories/CardRepository.cs [hunk 1]
```diff
@@ -0,0 +1,64 @@
...
+    {
+        return context.Cards.Find(id);
+    }
+
+    public bool CardExists(int id)
+    {
+        return context.Cards.Any(c => c.Id == id);
+    }
+
+    public bool AddCardToDeck(int deckId, string question, string answer)
```
Comment (rokas-migauskas) [inline aiKart/Repositories/CardRepository.cs:29]: null check nereiktu?

File: aiKart/Repositories/DeckRepository.cs [hunk 2]
```diff
@@ -0,0 +1,56 @@
...
+    private readonly DataContext context;
+
+    public DeckRepository(DataContext context)
+    {
+        this.context = context;
+    }
+ 
+    public ICollection<Deck> GetDecks()
+    {
+        return context.Decks.OrderBy(d => d.Id).ToList();      // later this could be modified to order be by last edit
```
Comment (rokas-migauskas) [inline aiKart/Repositories/DeckRepository.cs:18]: Turbut tiesiog galima bus turet atskirus metodus su skirtingu order. Kitas variantas turet dar viena enum pasirinkimu pagal ka orderint ir passint kaip argumenta.

File: aiKart/Repositories/CardRepository.cs (hunk 1 above)
Comment (K-Karalius) [reply aiKart/Repositories/CardRepository.cs:29]: man rodos best practise null value tikrint yra controller'iu klasese arba service'o klasese

File: aiKart/Repositories/DeckRepository.cs (hunk 2 above)
Comment (K-Karalius) [reply aiKart/Repositories/DeckRepository.cs:18]: Galima bus veliau prisidet papildomus metodus, kad tikrint pagal kitus properities. Bet ieskojimas pagal id, man rodos, yra greiciau nei pagal kitus.

Comment (rokas-migauskas) [reply aiKart/Repositories/DeckRepository.cs:18]: Viskas ok, bet šiaip komentarų palikinet nereiktų, nebent kažkoks //TODO: svarbus

**Comment by MartynasJonkus** [comment]
Could you organize the database files more clearly and separate them from the .cs files that will interact with the frontend

**Comment by K-Karalius** [comment]
> Could you organize the database files more clearly and separate them from the .cs files that will interact with the frontend

no

**Comment by MartynasJonkus** [comment]
> > Could you organize the database files more clearly and separate them from the .cs files that will interact with the frontend
> 
> no :)

>:(

**Pull request merged**